scriptmodules/emulators/dosbox-staging.sh

```diff
@@ -56,15 +51,23 @@ function configure_dosbox-staging() {
 
     [[ "$md_id" == "remove" ]] && return
 
+    # copy required resources
+    local config_dir="$md_conf_root/pc"
+    cp -R "$md_build/contrib/resources/"{freedos-cpi,freedos-keyboard,glshaders,translations} "$config_dir/"
```

Comment:
Shouldn't the resources be copied to `$md_inst/resources` ?
I've noticed `meson install` doesn't copy them, which is unfortunate.

Reply:
No. For that release everything to be copied over the "config" directory (which is `/home/$user/.dosbox` by default).  If that's an issue we would have to think about creating a sub-dir into .dosbox eg. `.dosbox/staging` or similar.

Reply:
According to https://github.com/dosbox-staging/dosbox-staging/blob/main/docs/README.resources, it should be possible to ship it as `$md_inst/resources/` (option 2. from the list). I'd prefer that resources shipped by the application to be copied in the install rather than to the user's folder. 

On my installation (for the previous version) I've used `$HOME/.config/dosbox/`'s folders (`glshader` and `mt32-roms`) for my local files, but I'll give it a test with the `$md_inst/resources` for the new resources.

Reply:
I stand corrected and I'm glad to catch this one up. I feel bad to be honest :( You're absolutely right and it's much better to have all those files under the "install dir" instead of .config. It's much cleaner plus it simplifies even further the script. Thanks again for that and apologize for not only missing that but for being wrong... 

Script edited accordingly.

Reply:
No worries, should be fine now.